src/OXRS_Rack32.cpp

```diff
@@ -443,6 +486,30 @@ void OXRS_Rack32::_initialiseEthernet(byte * mac)
     Serial.println(F("none"));
   }
 }
+#else
+void OXRS_Rack32::_initialiseWifi(byte * mac)
+{
+ // Get ESP32 base MAC address
+  WiFi.macAddress(mac);
+  
+  // Display the MAC address on serial
+  char mac_display[18];
+  sprintf_P(mac_display, PSTR("%02X:%02X:%02X:%02X:%02X:%02X"), mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
+  Serial.print(F("[ra32] wifi mac address: "));
```

Comment:
Maybe add `ethernet` to the logging on lines 463 and 479, for consistency?

Reply:
I had the same thought regarding enable both ast the same time for failover or runtime switchable option but didn't go further into that due to the coplexity. We can leave this branch until we have time to add the captive portal.